drivers/hmis/app/graphql/types/hmis_schema/client.rb

```diff
@@ -83,14 +83,38 @@ def self.authorized?(object, ctx)
     field :phone_numbers, [HmisSchema::ClientContactPoint], null: false
     field :email_addresses, [HmisSchema::ClientContactPoint], null: false
     field :hud_chronic, Boolean, null: true, description: 'Meets the definition for HUD chronically homeless as of today (time of API request)'
-    field :eligible_ce_opportunities, Types::HmisSchema::CeOpportunity.page_type, null: false
 
-    def eligible_ce_opportunities
+    field :eligible_ce_opportunities, Types::HmisSchema::CeOpportunity.page_type, null: false do
+      # Omit status, since we only return open opportunities for the client anyway
+      filters_argument Types::HmisSchema::CeOpportunity, omit: [:status], type_name: 'ClientEligibleCeOpportunity'
+    end
+    def eligible_ce_opportunities(filters: nil)
+      raise unless Hmis::Ce.configuration.enabled?
+
+      scope = Hmis::Ce::Opportunity.for_client(object)
```

Comment:
suggest wiring up the visibility scope
```suggestion
      scope = Hmis::Ce::Opportunity.viewable_by(current_user).for_client(object)
```